src/assets/i18n/en.json

```diff
@@ -292,9 +292,25 @@
       "RIGHT_HERE": "Right here",
       "TITLE": "Danger Sign"
     },
-    "DELETE": "Delete observation",
-    "DELETE_CONFIRM": "Are you sure?",
-    "DIRT": {
+    "DELETE": {
```

Comment:
Det blir litt vanskelig for oversetterne å skjønne forskjellen her tror jeg. Hva om vi sletter `DELETE.BUTTON`, og heller bruker `DELETE.DRAFT.BUTTON`, så det alltid står "Delete draft" for kladder, og endrer navnet på `REGOBS` til `SUBMITTED_REGISTRATION`?

Reply:
Enig. Blei ryddigere nå.